docs/components/modeler/desktop-modeler/process-applications.md

```diff
@@ -0,0 +1,55 @@
+---
+id: process-applications
+title: Process applications in Desktop Modeler
+description: In Desktop Modeler a process application is a folder that contains a .process-application file and a set of related files you can work on and deploy as a single bundle.
+---
+
+import GroupingImg from './process-applications/img/grouping.png'
+import OverlayImg from './process-applications/img/overlay.png'
+
+In Desktop Modeler a [process application](../../concepts/process-applications.md) is a folder that contains a `.process-application` file and a set of Camunda 8 assets that you work on and deploy as a single bundle. Typically, a process application also contains additional assets like job worker implementations.
```

Comment:
Don't re-define process applications here, instead link:

```suggestion
Desktop modeler recognizes process applications you build and offers you advanced editor intelligence, deployment and execution features within the context of such an application.

To identify the boundaries of a process application, Desktop Modeler searches for a `.process-application` file in the root of your project. [...]
```

(Highlight some of the features that work).

Reply:
We don't have deployment yet. Should we mention this already anyways?

Reply:
Don't mention as a feature what we don't have yet. But have that list as a structure we can append to.

As I look onto that page, I want to know "what does modeler offer to me to support process applications".